core/db_models/EEM_Form_Section.model.php

```diff
@@ -0,0 +1,306 @@
+<?php
+
+use EventEspresso\core\services\request\RequestInterface;
+
+/**
+ * Class EEM_Form_Section
+ * Model Fields:
+ *      FSC_ID int(10) unsigned NOT NULL AUTO_INCREMENT,
+ *      FSC_UUID binary(16) NOT NULL,
+ *      FSC_appliesTo varchar(255) NOT NULL,
+ *      FSC_belongsTo binary(16) NULL,
+ *      FSC_htmlClass text NULL,
+ *      FSC_order tinyint(2) unsigned NOT NULL DEFAULT 0,
+ *      FSC_relation tinytext NULL,
+ *      FSC_status tinytext NULL,
+ *      FSC_wpUser bigint(20) unsigned NULL,
+ *
+ * @author  Brent Christensen
+ * @since   $VID:$
+ */
+class EEM_Form_Section extends EEM_Base
+{
+    const APPLIES_TO_ALL         = 'all';
+
+    const APPLIES_TO_PRIMARY     = 'primary';
+
+    const APPLIES_TO_PURCHASER   = 'purchaser';
+
+    const APPLIES_TO_REGISTRANTS = 'registrants';
+
+    const STATUS_ACTIVE          = 'active';
+
+    const STATUS_ARCHIVED        = 'archived';
+
+    const STATUS_DEFAULT         = 'default';
+
+    const STATUS_TRASHED         = 'trashed';
+
+    /**
+     * @var EEM_Form_Section
+     */
+    protected static $_instance;
+
+    /**
+     * @var RequestInterface
+     */
+    private $request;
+
+    /**
+     * @var array
+     */
+    private $valid_applies_to_options;
+
+    /**
+     * @var array
+     */
+    private $valid_status_options;
+
+
+    protected function __construct($timezone = null)
+    {
+        $this->valid_applies_to_options = apply_filters(
+            'FHEE__EEM_Form_Section__valid_applies_to_options',
+            [
+                EEM_Form_Section::APPLIES_TO_ALL         => esc_html__('All Registrants', 'event_espresso'),
+                EEM_Form_Section::APPLIES_TO_PRIMARY     => esc_html__('Primary Registrant Only', 'event_espresso'),
+                EEM_Form_Section::APPLIES_TO_PURCHASER   => esc_html__('Purchasing Agent', 'event_espresso'),
+                eem_form_section::APPLIES_TO_REGISTRANTS => esc_html__('Additional Registrants', 'event_espresso'),
```

Comment:
```suggestion
                EEM_Form_Section::APPLIES_TO_REGISTRANTS => esc_html__('Additional Registrants', 'event_espresso'),
```

Reply:
what the...???  no idea how that happened... thnx